.travis.yml

```diff
@@ -0,0 +1,46 @@
+language: shell
+notifications:
+  email:
+    on_success: never
+stages:
+- tests
+- deployment
+env:
+  global:
+    - RELEASE_REPO_TO_FIND="https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/releases/download/latest/"
+    - RELEASE_REPO_TO_REPLACE="https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/releases/download/$TRAVIS_TAG/"
+jobs:
+  include:
+    - stage: tests
+      name: Tests & Linting
+      language: python
+      python:
+        - "3.8"
+      install:
+        - pip3 install -r src/requirements.txt
+        - pip3 install -r src/requirements-dev.txt
+      before_script:
+        - cd src
+      script:
+        - python3 -m pylint --rcfile=../pipeline/pylint.cfg $(find . -name "*.py")
+        - python3 -m pytest -v ../tests/unit
+    - stage: deployment
+      name: Github Release Deployment
+      if: tag =~ /^release.*$/
+      language: python
+      python:
+        - "3.8"
+      install: skip
+      # using 'script' instead of 'before_deploy' because it's unskippable in Python environment
+      # https://docs.travis-ci.com/user/languages/python/#default-build-script
+      script:
+        - chmod +x build-release-package.sh
+        - chmod +x version.sh
+        - ./build-release-package.sh # needs Python environment internally
+      deploy:
+        provider: releases
+        api_key: $GITHUB_RELEASE_API_KEY
+        file: ./dynatrace-aws-log-forwarder.zip

```

Comment:
I opted for a single release package (unlike Azure) because:

- It doesn't require CloudFormation template modifications in Travis, see: https://github.com/dynatrace-oss/dynatrace-azure-log-forwarder/blob/master/.travis.yml#L41
- It works even after repo rename or any other changes to the URL of the remote files
- `aws lambda update-function-code` doesn't support remote GH files, only local ones or S3